jina/helper.py

```diff
@@ -1163,12 +1162,18 @@ def dunder_get(_dict: Any, key: str) -> Any:
 
 if False:
     from fastapi import FastAPI
+    from jina.peapods.runtimes.asyncio.rest.models import JinaRequestModel
+    from starlette.responses import StreamingResponse
 
 
-def extend_rest_interface(app: 'FastAPI') -> 'FastAPI':
+def extend_rest_interface(
+    app: 'FastAPI',
+    client: Optional[Callable[['JinaRequestModel', str], 'StreamingResponse']] = None,
```

Comment:
I think `client` does not seem like an appropiate name.

Also the `type` hint seems like `missing` Unions?